Configurateur/front/lib/components/container.dart

```diff
@@ -7,6 +7,18 @@ import 'package:front/services/storage_service.dart';
 import 'package:front/styles/globalStyle.dart';
 import 'package:go_router/go_router.dart';
 
+/// ContainerListData
+///
+/// Define the data of container in back end
+/// [id] : Container's id
+/// [createdAt] : Creation of the container
+/// [organization] : Organization having created the container
+/// [organizationId] : Id of the organization
+/// [containerMapping] : ???
```

Comment:
???? Can not leave that undocumented

Reply:
it's a string that contains numbers representing where lockers is positioned in the container.